prow/spyglass/lenses/buildlog/lens.go

```diff
@@ -124,29 +131,37 @@ type BuildLogsView struct {
 	LogViews           []LogArtifactView
 	RawGetAllRequests  map[string]string
 	RawGetMoreRequests map[string]string
```

Comment:
I know you didn't add these, but it looks like they are unused. Can we remove them?